tests/test_appliancesmanager.py

```diff
@@ -0,0 +1,148 @@
+    http_client_mock: AiohttpClientMocker,
+):
+    http_client_mock.create_session(asyncio.get_event_loop())
+    auth = Auth(BACKEND_SELECTOR_MOCK, "email", "secretpass", http_client_mock.session)
+    auth._auth_dict["access_token"] = "acess_token_123"
+
+    am = AppliancesManager(BACKEND_SELECTOR_MOCK, auth, http_client_mock.session)
+
+    headers = am._create_headers()
+
+    assert headers["WP-CLIENT-BRAND"] == BACKEND_SELECTOR_MOCK.brand.name
+
+    await http_client_mock.close_session()
+
+
+async def test_account_id_set_if_populated_in_auth_dict(
+    http_client_mock: AiohttpClientMocker,
+):
+    # mock_appliancesmanager_get_account_id_get would set the acocunt_id to 12345
+    # so if _get_account_id returns 55555, that means we are using the provided value
+    # which is what we want
+    mock_appliancesmanager_get_account_id_get(http_client_mock, BACKEND_SELECTOR_MOCK)
+    http_client_mock.create_session(asyncio.get_event_loop())
+    auth = Auth(BACKEND_SELECTOR_MOCK, "email", "secretpass", http_client_mock.session)
+    auth._auth_dict["accountId"] = "555555"
+
+    am = AppliancesManager(BACKEND_SELECTOR_MOCK, auth, http_client_mock.session)
+
+    account_id = await am._get_account_id()
+    assert account_id == "555555"
+
+    await http_client_mock.close_session()
+
+
+async def test_fetch_appliances_calls_get_account_id_if_not_set(
+    http_client_mock: AiohttpClientMocker,
+):
+    mock_appliancesmanager_get_account_id_get(http_client_mock, BACKEND_SELECTOR_MOCK)
+
+    http_client_mock.create_session(asyncio.get_event_loop())
+    auth = Auth(BACKEND_SELECTOR_MOCK, "email", "secretpass", http_client_mock.session)
+
+    am = AppliancesManager(BACKEND_SELECTOR_MOCK, auth, http_client_mock.session)
+
+    account_id = await am._get_account_id()
+
+    assert account_id == "12345"
+    assert account_id != auth.get_account_id()
+
+    await http_client_mock.close_session()
+
+
+async def test_fetch_appliances_calls_both_get_methods(
+    http_client_mock: AiohttpClientMocker,
+):
+    mock_appliancesmanager_get_account_id_get(http_client_mock, BACKEND_SELECTOR_MOCK)
+    mock_appliancesmanager_get_owned_appliances_get(
+        http_client_mock, BACKEND_SELECTOR_MOCK, "12345"
+    )
+    mock_appliancesmanager_get_shared_appliances_get(
+        http_client_mock, BACKEND_SELECTOR_MOCK
+    )
+
+    http_client_mock.create_session(asyncio.get_event_loop())
+    auth = Auth(BACKEND_SELECTOR_MOCK, "email", "secretpass", http_client_mock.session)
+
+    am = AppliancesManager(BACKEND_SELECTOR_MOCK, auth, http_client_mock.session)
+    am._get_shared_appliances = get_mock_coro(True)
+    am._get_owned_appliances = get_mock_coro(False)
+
+    await am.fetch_appliances()
+
+    am._get_owned_appliances.assert_called()  # type: ignore
+    am._get_shared_appliances.assert_called()  # type: ignore
+    await http_client_mock.close_session()
+
+
+async def test_fetch_appliances_loads_lists(
```

Comment:
if we update the test above to use only the public methods, I think this can be merged into it, right?